lib/util.js

```diff
@@ -306,11 +306,22 @@ if (process.env.NODE_DEBUG) {
   debugEnvRegex = new RegExp(`^${debugEnv}$`, 'i');
 }
 
+// emits warning, when user sets
+// NODE_DEBUG=http.
+function emitWarningIfNeeded(set) {
+  if ('HTTP' === set) {
```

Comment:
This has to be guarded with a flag that we did not emit this warning already.
Else the following code would emit the warning twice:
```js
require('util').debuglog('http')('foo')
require('util').debuglog('http')('foo')
```